framework/transport/ieee1905_transport/include/mapf/transport/ieee1905_transport.h

```diff
@@ -143,8 +144,8 @@ class Ieee1905Transport {
         unsigned int dst_if_index = 0;
         uint8_t src_if_type       = CmduRxMessage::IF_TYPE_NONE;
         unsigned int src_if_index = 0;
-        uint8_t dst[ETH_ALEN]     = {0}; // destination mac address
-        uint8_t src[ETH_ALEN]     = {0}; // source mac address
+        sMacAddr dst              = {.oct = {0}}; // destination mac address
```

Comment:
Maybe you should also move ZERO_MAC from network_utils to TLVF

Reply:
True, but there if I do this I can also move a lot more. I prefer to keep this for a separate PR.